test/ics02-wrapper/ICS02PrecompileWrapperTest.t.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+// solhint-disable gas-custom-errors
+
+import { Test } from "forge-std/Test.sol";
+
+import { ILightClientMsgs } from "../../contracts/msgs/ILightClientMsgs.sol";
+import { IICS02ClientMsgs } from "../../contracts/msgs/IICS02ClientMsgs.sol";
+
+import { ICS02PrecompileWrapper } from "../../contracts/light-clients/ics02-wrapper/ICS02PrecompileWrapper.sol";
+import {
+    IICS02PrecompileWrapperErrors
+} from "../../contracts/light-clients/ics02-wrapper/errors/IICS02PrecompileWrapperErrors.sol";
+import { IICS02Precompile } from "../../contracts/light-clients/ics02-wrapper/interfaces/IICS02Precompile.sol";
+
+contract ICS02PrecompileWrapperTest is Test {
+    address public constant ICS02_ADDRESS = 0x0000000000000000000000000000000000000807;
+
+    string public constant TEST_CLIENT_ID = "07-tendermint-0";
+    ICS02PrecompileWrapper public ics02Wrapper;
+
+    function setUp() public {
+        ics02Wrapper = new ICS02PrecompileWrapper(TEST_CLIENT_ID);
+    }
+
+    function test_success_goClientId() public view {
+        assertEq(ics02Wrapper.ibcGoClientId(), TEST_CLIENT_ID);
+    }
+
+    function test_success_getClientState() public {
```

Comment:
We's also want to test these on an actual chain - lots of mocks here.

Reply:
1. Yes, I'll add e2es.
2. This contract doesn't even know what the chain is. So, this suffices to see if this contract does its job. E2es will make it clear if the job it does is what we need it to do